<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registrar</class>
 <widget class="QWidget" name="Registrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>REGISTRO</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_img">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#l_img{
border:1px solid white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_registrar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cargar_foto">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Foto</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>291</width>
     <height>183</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_nombre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_apellido"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cedula</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txt_cedula"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Rol</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comb_rol">
      <item>
       <property name="text">
        <string>ESTUDIANTE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOCENTE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COORDINADOR</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Carrera</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comb_carrera">
      <item>
       <property name="text">
        <string>ADMINISTRACIÓN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AGROALIMENTARIA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DERECHO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>INFORMÁTICA</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
